lbrynet/daemon/Daemon.py

```diff
@@ -163,6 +163,11 @@ def __call__(self):
         return d
 
 
+def sort_claim_results(claims):
+    claims.sort(key=lambda d: (d['height'], d['name'], d['claim_id'], d['txid'], d['nout']))
+    return claims
```

Comment:
This doesn't need to return, the provided `claims` list is sorted in place.

Reply:
`sort_claim_results` is provided as a Deferred callback so if it doesn't return then the subsequent callbacks will not get the claims.

Reply:
Ok, makes sense